translations/pl.json

```diff
@@ -6,7 +6,7 @@
         },
         "error": {
             "auth_error": "Błąd połączenia, sprawdź token",
```

Comment:
As there is not token set on setup page, i would rename it to EUID
```suggestion
            "auth_error": "Błąd połączenia, sprawdź EUID",
```

Reply:
Sure.